modules/installation-restricted-network-azure-firewall.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-restricted-networks-azure-installer-provisioned.adoc
+
+:_content-type: CONCEPT
+[id="installation-restricted-network-azure-firewall_{context}"]
+= About creating a cluster in a restricted network with Azure Firewall
+
+You can use Azure Firewall to restrict the outbound routing for the Virtual Network (VNet) that is used to install the {product-title} cluster. For more information, see link:https://learn.microsoft.com/en-us/azure/aks/egress-outboundtype#deploy-a-cluster-with-outbound-type-of-udr-and-azure-firewall[providing user-defined routing with Azure Firewall]. You can create a {product-title} cluster in a restricted network by using VNet with Azure Firewall and configuring the user-defined routing.
```

Comment:
This seems related to your module on user-define outbound routing. Is this the only option to control outbound routing for a cluster or simply one way of going about it?

Reply:
This is one of restricting outbound routing. The other one option is to use network security groups as explained here: https://github.com/openshift/openshift-docs/pull/61086/files#diff-6987ad7912fc82b29a28ebdf544d03faefabdf030c6cbf15903dbb8865110a9eR110-R129